config/hop.json

```diff
@@ -193,35 +246,5 @@
         "ammWrapper": "0x6329d42Cc5517ba1d913303E019a556d84B5a05d"
       }
     }
```

Comment:
Can you please also update the testnet configs based on this file:
https://github.com/hop-protocol/hop/blob/develop/packages/core/src/addresses/goerli.ts

Reply:
@maxklenk updated